src/ast.rs

```diff
@@ -1766,4 +1808,86 @@ event zeek_init() { for (v in vs) ; }
         let typ = db.typ(decl).unwrap();
         assert_debug_snapshot!(typ);
     }
+
+    #[test]
+    fn loop_vars_set_iteration_order() {
+        let mut db = TestDatabase::default();
+        let uri = Arc::new(Url::from_file_path("/x.zeek").unwrap());
+        db.add_file(
+            (*uri).clone(),
+            r#"
+global vs: set[string, count] = set("a", 1);
```

Comment:
The RHS is invalid Zeek, I think this would need to be something like `{ [1, "one"] }`.